packages/@dcl/inspector/src/components/Tree/Tree.tsx

```diff
@@ -284,10 +290,38 @@ export function Tree<T>() {
         }
       }
 
-      const isEntity = useMemo(() => {
+      const isValidEntity = useMemo(() => {
         return typeof value !== 'string'
       }, [value])
 
+      const isEntity = useMemo(() => {
+        if (typeof value === 'string') return false
+        return value !== ROOT && value !== PLAYER && value !== CAMERA
+      }, [value])
```

Comment:
this should be a check in the useTree hook. The Tree component should not know about `ROOT,etc` entities

Reply:
there is an `isRoot` function in the `useTree` hook that we can use 😋